homeassistant/components/influxdb.py

```diff
@@ -114,7 +114,8 @@ def influx_event_listener(event):
 
         for key, value in state.attributes.items():
             if key != 'unit_of_measurement':
-                json_body[0]['fields'][key] = value
+                if isinstance(value, (str, int, float)):
```

Comment:
Booleans do work and added. I also had to make a change to convert all ints to floats. If the data changes in HA python sends it as int or float. This causes errors if a previous record was a different data type.